tools/krona/tool_dependencies.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<tool_dependency>
+	<package name="perl" version="5.18.1">
+        <repository name="package_perl_5_18" owner="iuc" prior_installation_required="True" />
+    </package>
+
```

Comment:
Perhaps a part of the confusion is the decision to put deps into tools-uic and actual tools into tools-devteam. Seems a bit too complicated for me though